dotnet-docs/docker-installation.md

```diff
@@ -42,13 +56,19 @@ This article is a step-by-step tutorial on deploying Telerik Report Server for .
 1. Run the command `docker stack deploy -c docker-compose.yml report-server`.
 1. Navigate to `localhost:82` in the browser to open the Report Server Manager for .NET.
 
->important The first time you open the Report Server, you will to configure it as explained in the article [Application Startup]({%slug application-startup%}).
 
-### Creating new Server Agent
+>important The first time you open the Report Server, you need to configure it as explained in the article [Application Startup]({%slug application-startup%}).
+
+We recommend using the images we provide instead of building custom ones. The sample code in the following sections assumes you use the images we provide. If you prefer custom images, change the code as follows:
+
+* `image: progressofficial/telerik-reportserver-app:latest` to `image: telerik-report-server:local`
+* `image: progressofficial/telerik-reportserver-agent:latest` to `image: telerik-report-server-agent:local`
+
+## Creating a new Server Agent
```

Comment:
~~I believe this is creating additional server agents? The current name suggests the installation up until now did not install a report agent at all, which i believe is not the case. Up in the section explain the result of the installation, e.g., you now have running RS app and running RS agent on the same docker container, etc.~~
Right, a default agent is NOT installed with the previous actions, then explain this in the conceptual part up in the article - that the installation process consists of two parts - first intalling and configuring the app, and then installing one or more agents. More are installed for scalability